azure-pipelines.yml

```diff
@@ -30,37 +31,37 @@ jobs:
   - script: echo semVer $(setVersionStep.semVer)
     name: echovar
 
-- job: build
-  displayName: Build database project & publish dacpac
-  pool:
-    vmImage: windows-2019
-  steps:
-  - task: VSBuild@1
-    displayName: 'Build database solution'
-    inputs:
-      solution: src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/NHSD.BuyingCatalogue.Ordering.OrderingDatabase.sqlproj
-
-  - task: CopyFiles@2
-    displayName: 'Copy Files to: $(build.artifactstagingdirectory)/database'
-    inputs:
-      SourceFolder: src
-      Contents: '**\*.dacpac'
-      TargetFolder: '$(build.artifactstagingdirectory)/database'
-      CleanTargetFolder: true
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: database'
-    inputs:
-      PathtoPublish: '$(build.artifactstagingdirectory)/database'
-      ArtifactName: database
-
-  - task: colinsalmcorner.colinsalmcorner-buildtasks.dacpac-report-task.DacPacReport@1
-    displayName: 'DacPac Schema Compare NHSD.BuyingCatalogue.Ordering.OrderingDatabase.dacpac'
-    inputs:
-      dropName: database
-      dacpacName: NHSD.BuyingCatalogue.Ordering.OrderingDatabase
-      targetDacPacPath: '$(build.artifactstagingdirectory)/database'
-    enabled: false
+# - job: build

```

Comment:
A whole lot of stuff commented out in this file, can I get a quick summary?

Reply:
Apologies, this was me testing out the pipeline to try and getting it working with the NuGet.config file. Turns out if I move it to the root of the project it works fine. So have made that change instead.